### PR TITLE
AB#2931 Intake start page

### PR DESCRIPTION
**Pull request by olivierroyESDC**
### Description
Added the start page for the intake form / process to enroll in cdcp

### Related Azure Boards Work Items
[AB#2931](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2931)

### Checklist

- [ ] I have tested the changes locally
- [ ] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [x] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [ ] I have checked that all e2e tests pass by running `npm run test:e2e`

### CODE REVIEW

File: frontend/app/routes/_gcweb-app.apply.apply-now.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+export default function ApplyNow() {
+  const { t } = useTranslation(i18nNamespaces);
+
+  return (
+    <>
+      <p className="mb-8 text-lg text-gray-500">{t('intake-forms:apply-form.privacy-statement')}</p>
+      <p>{t('intake-forms:apply-form.privacy-message')}</p>
+      <p />
```
Comment (gregory-j-baker) [inline frontend/app/routes/_gcweb-app.apply.apply-now.tsx:25]: You shouldn't use an empty paragraph. It isn't semantic HTML.

File: frontend/app/routes/_gcweb-app.apply.apply-now.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <p className="mb-8 text-lg text-gray-500">{t('intake-forms:apply-form.identity-information')}</p>
+      <p>{t('intake-forms:apply-form.identity-message')}</p>
+      <p />
```
Comment (gregory-j-baker) [inline frontend/app/routes/_gcweb-app.apply.apply-now.tsx:28]: You shouldn't use an empty paragraph. It isn't semantic HTML.

File: frontend/app/routes/_gcweb-app.apply.apply-now.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      <p className="mb-8 text-lg text-gray-500">{t('intake-forms:apply-form.terms-and-conditions')}</p>
+      <p>{t('intake-forms:apply-form.terms-and-conditions-message')}</p>
+      <p />
```
Comment (gregory-j-baker) [inline frontend/app/routes/_gcweb-app.apply.apply-now.tsx:31]: You shouldn't use an empty paragraph. It isn't semantic HTML.

Comment (nicholas-ly) [inline frontend/app/routes/_gcweb-app.apply.apply-now.tsx:31]: Avoid using empty `<p />` if the intent is to add extra spacing. Use CSS instead (maybe a `mb-8` like above?).

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]

File: frontend/app/routes/_gcweb-app.apply.apply-now.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import { getRaoidcService } from '~/services/raoidc-service.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('intake-forms');
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const raoidcService = await getRaoidcService();
+  await raoidcService.handleSessionValidation(request);
+
+  return '';
```
Comment (nicholas-ly) [inline frontend/app/routes/_gcweb-app.apply.apply-now.tsx:15]: **Suggestion/Nitpick:** Returning an empty string seems odd. An example from the Remix docs does this instead 
```javascript
export async function loader({ request }: LoaderFunctionArgs) {
  return json({ ok: true });
};
```

See https://remix.run/docs/en/main/route/loader

File: frontend/app/routes/_gcweb-app.apply.apply-now.tsx (hunk 1 above)
Comment (olivierroyESDC) [reply frontend/app/routes/_gcweb-app.apply.apply-now.tsx]: done

File: frontend/app/routes/_gcweb-app.apply.apply-now.tsx (hunk 3 above)
Comment (olivierroyESDC) [reply frontend/app/routes/_gcweb-app.apply.apply-now.tsx]: done

Comment (olivierroyESDC) [reply frontend/app/routes/_gcweb-app.apply.apply-now.tsx]: done

File: frontend/app/routes/_gcweb-app.apply.apply-now.tsx (hunk 4 above)
Comment (olivierroyESDC) [reply frontend/app/routes/_gcweb-app.apply.apply-now.tsx]: done

**Pull request merged**